### ISSUE TITLE
New-SSHSession, -AcceptKey doesn't accept $true

### ISSUE THREAD

**Issue by Jim-Wright** [darkoperator/Posh-SSH#101]
Version 1.7.5 installed.  Discovered that when I use the -AcceptKey parameter, setting this to $true per the documentation produces the following error:

PS C:\Users\wrightj> New-SSHSession -ComputerName "172.25.254.101" -Credential $credentials -AcceptKey $true
New-SSHSession : A positional parameter cannot be found that accepts argument 'True'.
At line:1 char:1
+ New-SSHSession -ComputerName "172.25.254.101" -Credential $credential ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [New-SSHSession], ParameterBindingException
    + FullyQualifiedErrorId : PositionalParameterNotFound,SSH.NewSshSession
 

Leaving the $true off doesn't produce an error (seems like it should?), and -AcceptKey seems to be defaulting to TRUE when used.

**Comment by darkoperator** [comment]
it is defined in the code as a switch, will try to find out why PowerShell parses it as a parameter that takes a value 

```
private bool _force = false;
        [Parameter(Mandatory = false,
            ValueFromPipelineByPropertyName = true,
            ParameterSetName = "Key")]
        [Parameter(Mandatory = false,
            ValueFromPipelineByPropertyName = true,
            ParameterSetName = "NoKey")]
        public SwitchParameter Force
        {
            get { return _force; }
            set { _force = value; }
        }
```

**Comment by BornToBeRoot** [comment]
Because your syntax is wrong:

Switch parameter are look like this `-AcceptKey:$true` not like `-AcceptKey $true`

![image](https://cloud.githubusercontent.com/assets/16019165/17221292/93336d32-54f3-11e6-84f8-993c2650c2bb.png)

**Comment by darkoperator** [comment]
Actually it is also a bad help file, some how I overwrote the old XML

**Comment by darkoperator** [comment]
pushed a fixed help file, will soon push a 1.7.6 to the gallery
